src/environments/environment.ts

```diff
@@ -1,5 +1,5 @@
 export const environment = {
   production: false,
-  //baseURL: 'https://localhost:7282/api/',
-  baseURL: 'http://aptitudetest-backend.web2.anasource.com/api/',
```

Comment:
don't commit environment file and use web hosted backend as base url

Reply:
okk